docs-v2/guides/webhooks.mdx

```diff
@@ -18,9 +18,9 @@ The webhook from Nango is a POST request with the following body:
     "providerConfigKey": "<integration-id>",  // The name of the integration in Nango, e.g. github
     "syncName": "<sync-name>",                // The name of the sync as defined in nango.yaml, e.g. github-tickets
     "model": "<data-model>",                  // The name of the data model, e.g. ticket
-    "responseResults": { "<DataModel>": { "added": 123, "updated": 123, "deleted": 123 } }, // The number of new/updated records, per object type. Only if records were delted will that property show.
+    "responseResults": { "<DataModel>": { "added": 123, "updated": 123, "deleted": 123 } }, // The number of new/updated/deleted records, per object type.
     "syncType": "INITIAL" | "INCREMENTAL",   // "Initial" or "incremental" run
-    "queryTimeStamp": "2023-05-31T11:46:13.390Z", // Use this with "delta" in getRecords to fetch the data of this sync
+    "queryTimeStamp": "2023-05-31T11:46:13.390Z", // Use this with "delta" in getRecords to fetch the data of this sync. Note that this property only exists for incremental syncs.
```

Comment:
This comment might be a bit misleading, because `incremental` is often opposed to `full refresh`. And we will send the queryTimeStamp for full refreshes. 

Maybe a way to phrase it is: Note that this property will be omitted (maybe a better DX here would be to make it `null`?) for the initial sync job (i.e. the 1st time the sync runs).

Reply:
Agree that a better DX is for it to be `null`.

> Note that this property will be null for the initial sync job.

Thoughts?